kaldb/src/main/proto/schema.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package slack.proto.kaldb.schema;
+
+option java_package = "com.slack.kaldb.proto.schema";
+
+// We want to call this Schema and then remove the .schema package
+// But if we do that today, the field types interfere with existing enums
+// which we don't want addressing just yet ( kaldb_search.proto )
+message PreprocessorSchema {
```

Comment:
Since this is consumed by both indexer and preprocessor, maybe "IngestSchema" is a better fit? Up to you.